nlds/rabbit_connection.py

```diff
@@ -23,30 +26,30 @@ def __init__(self, config: str = RABBIT_CONFIG_FILE_LOCATION):
     def get_connection(self):
         if self.connection is None or not (self.connection.is_open and self.channel.is_open):
             # Get the username and password for rabbit
-            rabbit_user = self.conf.get('server', 'rabbit_user')
-            rabbit_password = self.conf.get('server', 'rabbit_password')
+            rabbit_user = self.config["user"]
+            rabbit_password = self.config["password"]
 
             # Start the rabbitMQ connection
             connection = pika.BlockingConnection(
                 pika.ConnectionParameters(
-                    self.conf.get('server', 'rabbit_server'),
+                    self.config["server"],
                     credentials=pika.PlainCredentials(rabbit_user, rabbit_password),
-                    virtual_host=self.conf.get('server', 'rabbit_vhost'),
-                    heartbeat=300
+                    virtual_host=self.config["vhost"],
+                    heartbeat=60
                 )
             )
 
```

Comment:
What happens here if the connection to the exchange fails?

Reply:
This is a really good point, and sent me down a rabbit hole looking at how to do this a bit more robustly. For now I can add some try-catches to catch specific errors but I think this will need to be expanded significantly – I'll make an issue for it

Reply:
Will be addressed in #6